server/src/main/java/com/android/identity/wallet/server/VerifierServlet.kt

```diff
@@ -980,7 +980,12 @@ class VerifierServlet : HttpServlet() {
             encryptedJWT.decrypt(decrypter)
 
             val vpToken = encryptedJWT.jwtClaimsSet.getClaim("vp_token") as String
-            session.deviceResponse = vpToken.fromBase64Url()
+            if (session.requestType.format == DocumentFormat.MDOC) {
```

Comment:
In case we ever add additional formats to DocumentFormat, we may want to do this as a when() expression (and explicitly handle both cases), so the compiler will let us know if a new enum was added that we aren't handling.

Reply:
Discussed offline. Defaulting to no encoding is fine.